transmart-rest-api/grails-app/controllers/org/transmartproject/rest/QueryController.groovy

```diff
@@ -150,8 +163,7 @@ class QueryController extends AbstractQueryController {
     }
 
     /**
-     * High dimensional endpoint:
-     * <code>/v2/high_dim?assay_constraint=${assays}&biomarker_constraint=${biomarker}&projection=${projection}</code>
+     * Helper function for retrieving high dimensional hypercube data.
```

Comment:
The example call is still there, I moved it to the documentation of `observation()`.